lib/components/link.vue

```diff
@@ -16,7 +16,7 @@
     export default {
         computed: {
             componentType() {
-                return (this.$router && this.to) ? 'router-link' : 'a';
+                return (this.$router && this.to && !this.disabled) ? 'router-link' : 'a';
```

Comment:
It makes one potential bug. router-link automatically assigns active class to current route so when item is disabled it will loose active class.